contributing/google-summer-of-code-2018/README.md

```diff
@@ -176,11 +176,12 @@ Interested students are also encouraged to interact with our contributor communi
 
 - **Desirable Skills:** Familiarity with JavaScript and Python development.
 
-### Buttons and Cards UI
+### Rich Messages and Payloads
```

Comment:
I think we should keep buttons and cards reference in the title. Like `Rich Messages and Payloads for Buttons and Cards`

Reply:
@diegodorgam sorry looks good to me and I've been meaning to action this but I just saw it got pulled without that. Should we do another PR to change it as you suggested?